From: Dejan Latinovic <Dejan.Latinovic@imgtec.com>
Subject: flashrom FTBFS on mipsel
Date: Thu, 15 Jan 2015 16:12:23 +0000

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=775428

This change is available upstream, along with other changes.
With this patch I was able to build flashrom for mipsel locally.

Here is the commit that contains my fix.
https://github.com/stefanct/flashrom/commit/138245ea51472211ec12e7b4d176591f3f18ce38

Index: flashrom-0.9.7+r1852/internal.c
===================================================================
--- flashrom-0.9.7+r1852.orig/internal.c
+++ flashrom-0.9.7+r1852/internal.c
@@ -327,12 +327,12 @@ int internal_init(void)
 	/* Probe unconditionally for ITE Super I/O chips. This enables LPC->SPI translation on IT87* and
 	 * parallel writes on IT8705F. Also, this handles the manual chip select for Gigabyte's DualBIOS. */
 	init_superio_ite();
-#endif
 
 	if (board_flash_enable(board_vendor, board_model, cb_vendor, cb_model)) {
 		msg_perr("Aborting to be safe.\n");
 		return 1;
 	}
+#endif
 
 #if defined(__i386__) || defined(__x86_64__) || defined (__mips)
 	register_par_master(&par_master_internal, internal_buses_supported);
